12_listcomprehension_lab/01.py

```diff
@@ -1,8 +1 @@
-"""
-Use range() and list comprehension to get
-the list of all lowercase english letters
-Hint: look for chr() and ord()
-"""
-
-
-
+map(chr, range(ord('a'), ord('z')+1))
```

Comment:
ok
so what is it?
looks similar to reduce function..

2016-07-24 22:53 GMT+03:00 Gabor Szabo <notifications@github.com>:

> In 12_listcomprehension_lab/01.py
> <https://github.com/ynonp/python-examples-verint-2016-07/pull/197#discussion_r71994293>
> :
>
> > @@ -1,8 +1 @@
> > -"""
> > -Use range() and list comprehension to get
> > -the list of all lowercase english letters
> > -Hint: look for chr() and ord()
> > -"""
> > -
> > -
> > -
> > +map(chr, range(ord('a'), ord('z')+1))
>
> That's a solution, but that's not a list comprehension.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ynonp/python-examples-verint-2016-07/pull/197/files/3d36cc455cd507df8e81b2805f9e0debe06c31b7#r71994293>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATWrAOmIkmPF7CprgXvwUnMkqx4iXPgdks5qY8LPgaJpZM4JTm_w>
> .
>